sympy/matrices/determinant.py

```diff
@@ -328,8 +328,12 @@ def _adjugate(M, method="berkowitz"):
     return M.cofactor_matrix(method=method).transpose()
 
 
+def _charpoly_DOM(DOM):
+    return DOM.charpoly()
```

Comment:
This function hardly seems necessary.

Reply:
I also felt the same, but I let it there assuming if we had to add some other functionalities in it, in the future.
Will remove it for now.
Thanks.